testing/trafficserver/APKBUILD

```diff
@@ -0,0 +1,162 @@
+# Contributor: tcely <trafficserver+aports@tcely.33mail.com>
+# Maintainer: tcely <trafficserver+aports@tcely.33mail.com>
+_upstream="trafficserver"
+pkgname="${_upstream}"
+pkgver="8.0.5"
+pkgrel=0
+pkgdesc="high-performance building block for cloud services"
+url="https://trafficserver.apache.org"
+arch="x86_64"
```

Comment:
Please add an option as to why this is only available on x86_64

Reply:
64-bit only and `aarch64` is broken.